### PR TITLE
Fix pdf ballot logo.

### LINKED ISSUE #1998: Two problems with pdf export ballot papers
Description: Two problems appear with pdf ballot papers.

1. The PDF export of ballot papers didn't work, if the PDF file contains an image file. By clicking the ballot paper button, the PDF loading bar loads, but didn't produce a pdf file. It seems that the image file isn't correctly linked. 

2. Addtionally it is not possible to deselect the check box PDF BALLOT PAPER LOGO of an image file, if the image file is selected as PDF headerlogo (left) and PDF ballot paper logo. 

Image file for case 1:
Screenshot_20230202_115423

Reproduction of case 1:
1. open a meeting
2. open _files_ and upload a small logo image. set this image as **PDF ballot paper logo**
3. open _elections_
4. create a new election via plus-icon
5. add one candidate and create a new ballot by clicking _New ballot_. Choose non-nominal as voting type. the other atributes doesn't matter. 
6. click the three-dot-menu in new ballot and choose Ballot papers.

Reproduction of case 2:
1. open a meeting
2. open files and upload a small logo image. set this image as PDF headerlogo (left) and PDF ballot paper logo.
3. try to deselect PDF ballot paper logo.

What should happen:
- [ ] case 1: pdf export of ballot papers with logo should be possible
- [ ] case 2: deselection should work correctly

### PR DESCRIPTION
**Pull request by m-schieder**
Fix #1998

### CODE REVIEW

**Review by luisa-beerboom — APPROVED** [review]

**Comment by emanuelschuetze** [comment]
I updated margin (left) to print ballot circle left-justified with title. And reduced left margin of assignemt ballots.

**Review by emanuelschuetze — APPROVED** [review]

File: client/src/app/gateways/export/pdf-document.service/pdf-document.service.ts [hunk 1]
```diff
@@ -435,6 +435,15 @@ export class PdfDocumentService {
         createVfs: () => Promise<PdfVirtualFileSystem>
     ): void {
         this.showProgress();
+
+        var logoBallotPaperUrl = this.mediaManageService.getLogoUrl(`pdf_ballot_paper`);
+        if (logoBallotPaperUrl) {
+            if (logoBallotPaperUrl.indexOf(`/`) === 0) {
+                logoBallotPaperUrl = logoBallotPaperUrl.substr(1); // remove trailing /
```
Comment (jsangmeister) [inline client/src/app/gateways/export/pdf-document.service/pdf-document.service.ts:442]:
```suggestion
                logoBallotPaperUrl = logoBallotPaperUrl.substr(1); // remove leading `/`
```

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Review by luisa-beerboom — APPROVED** [review]

**Review by jsangmeister — APPROVED** [review]

**Comment by m-schieder** [comment]
@emanuelschuetze  can you look at it again. The PDF should look exactly the same as after your changes and we now have no negative margin.

**Review by emanuelschuetze — APPROVED** [review]
Looks good now. - I just fixed the margin for YNA ballots.

**Pull request merged**